mlflow/spark.py

```diff
@@ -513,17 +530,50 @@ def save_model(
     )
 
 
-def _load_model(model_uri, dfs_tmpdir=None):
+def _shutil_copytree_without_file_permissions(src_dir, dst_dir):
+    """
+    Copies the directory src_dir into dst_dir, without preserving filesystem permissions
+    """
+    for (dirpath, dirnames, filenames) in os.walk(src_dir):
+        for dirname in dirnames:
+            relative_dir_path = os.path.relpath(os.path.join(dirpath, dirname), src_dir)
+            # Compute corresponding FUSE path of each local directory and create an equivalent
+            # FUSE directory
+            fuse_dir_path = os.path.join(dst_dir, relative_dir_path)
```

Comment:
This is no longer specific to fuse, it's just "absolute_dir_path" right?

Reply:
👍 yep that's right! I also added a unit test for this method, although it's an internal one used only in the `mlflow.spark` module, because it's somewhat complex